.travis.yml

```diff
@@ -96,6 +98,9 @@ script:
       petastorm/tests/test_pytorch_dataloader.py
       petastorm/tests/test_pytorch_utils.py
 
+    # Run test_tf_autograph separately because the error message "AutoGraph could not transform..." only show once, so we need ensure no other TF tests run before it.
+  - $RUN $PYTEST petastorm/tests/test_tf_autograph.py
```

Comment:
Please add `-v` (actually, can move `-v` into PYTEST= definition), `-Y` and `--cov-append` to this invocation (this is the reason, I think, for all-red codecov).

Reply:
-v was already in PYTEST definition